### ISSUE TITLE
Lab Report 1 Feedback

### ISSUE THREAD

**Issue by MohaElder** [ryanp0126/cse15l-lab-reports#1]
Here is your feedback for Lab Report 1:

- [x] All relevant URLs written as markdown links
     - https://sdacs.ucsd.edu/~icc/index.php this should also be a link
- [x] All relevant code examples written with backticks (```)
- [x] All screenshots correspond to the described steps
- [x] All titles of sections are formatted as headers or similar
  - `` ## Setting an SSH key `` this seems not working
- [x] Lab report is on its own page, not the index page
- [x] Last example counts keystrokes and shows how to run remotely saving time
 - refer to [lab report week1](https://ucsd-cse15l-w22.github.io/week/week1/#part-7-making-remote-running-even-more-pleasant) and add keystrokes part
- [x] Text is formatted in a way that was easy for grader to read

Additional comments from your grader:

Good job on your lab report! The problems are all minor, but can be perfect if you fix them. Keep it going!

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by ryanp0126** [comment]
Hi, I just added the changes to my lab report - do I need to do anything else like submit on gradescope or something?

**Comment by MohaElder** [comment]
Yes, you need to submit a regrade request just as the issue has mentioned.
